edpop_explorer/srumarc21reader.py

```diff
@@ -89,9 +94,17 @@ def __repr__(self):
 
 
 class SRUMarc21Reader(SRUReader):
-    def _convert_record(self, sruthirecord: dict) -> APIRecord:
+    marcxchange_prefix = ''
+    records: List[Marc21Record]
+
+    def _convert_record(self, sruthirecord: dict) -> Marc21Record:
```

Comment:
There's a lot happening in this method, also with many long expressions. Do you see any opportunities for adding more comments and/or outfactoring parts of it? Just asking, not a hard condition for approval.

Reply:
I outfactored the most confusing part to a different method and I better explained what is going on.